.test-infra/dataproc/init-actions/docker.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
This file was copied from: https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/docker/docker.sh and was unchanged.

Is it ok to copy this like that? What license should I put there?

Reply:
It looks like this project is licensed under the Apache license, so it should be fine to just add it. I'll add it for you now to get the precommits running (even though I guess they won't exercise this code..)

Reply:
Ok. Thanks!